examples/graphviz_layout/plot_atlas.py

```diff
@@ -14,40 +18,21 @@
 import networkx as nx
 
 
+GraphMatcher = nx.isomorphism.vf2userfunc.GraphMatcher
+
+
 def atlas6():
-    """Return the atlas of all connected graphs of 6 nodes or less.
-    Attempt to check for isomorphisms and remove.
-    """
+    """Return the atlas of all connected graphs with at most 6 nodes"""
 
-    Atlas = nx.graph_atlas_g()[0:208]  # 208
-    # remove isolated nodes, only connected graphs are left
+    Atlas = nx.graph_atlas_g()[3:209]  # 0, 1, 2 => no edges. 208 is last 6 node graph
```

Comment:
This is fine for now, but a) what does `_g` mean in the naming of that function and b) should we not have a more generic way of querying graphs in the atlas in a certain size range?  `graph_atlas_by_nodes(min=0, max=6)` or something like that?

Reply:
Nice idea -- and should be a different PR so thanks.

The `_g` was added well before the first release of NX and probably not thought about much since.
It designates that it is returns the full list of all graphs with 7 or fewer nodes -- as opposed to `nx.graph_atlas(23)` which is a function that creates one graph from the atlas.

Perhaps a better name would be `nx.graph_atlas_list`.  
Or maybe combining the two and having the function with no input return the entire list. 
I prefer the latter.

I haven't thought much about what kind of queries users would want from this atlas. As the docstring suggests, the Atlas itself sorts by number of nodes, number of edges and lexicographic degree sequence order. But there are 3 mistakes in that ordering by the atlas. We left the mistakes in to match the Atlas. It might be worth revisiting that too.

Potential headaches with `nx.graph_atlas(min_nodes=2, max_nodes=6, min_edges=3, max_edges=10, no_degree_less_than=2)`   :) Maybe we draw the line somewhere useful yet simple...  I would say either no such option or only restrict by number of nodes.  

Workaround:   `[G for G in nx.graph_atlas_g() if len(G)>=2 if len(G)<=6]` 
or any other conditions you want to put on G...